_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,118 @@
+---
+id: b4f0c3
+name: meta viewport
+rule_type: atomic
+description: |
+  This rule checks that the meta element is not used to block the user agent ability to zoom 
+accessibility_requirements:
+  wcag20:1.4.10: # Reflow (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to every `<meta name="viewport">` elements with a `content` attribute in a document.
+
+## Expectation
+
+The `content` attribute of the test target: 
+- do not have the property `user-scalable` or it is set to `yes`
+- do not have the property `maximum-scale` 
+
+## Assumptions
+
+- There may be some inconsistencies with how different browser vendors interpret `<meta name="viewport">`.
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+## Background
+
+- [The `meta` element](https://www.w3.org/TR/html52/document-metadata.html#the-meta-element)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The meta `viewport` do not define the `maximum-scale` and `user-scalable` properties.
```

Comment:
do > does

Shouldn't `meta` be written between backticks because you're referring to the HTML element? If so, change in all examples, please.